runelite-client/src/test/java/net/runelite/client/plugins/slayer/SlayerPluginTest.java

```diff
@@ -86,6 +86,8 @@
 	private static final String TASK_KONAR_BOSS = "You're now assigned to bring balance to the Alchemical<br>Hydra 35 times. Your reward point tally is 724.";
 
 	private static final String TASK_EXISTING = "You're still hunting suqahs; you have 222 to go. Come<br>back when you've finished your task.";
+	private static final String TASK_EXISTING_2 = "You're still bringing balance to basilisks in the Jormungand's Prison, with 136 to go. Come back when you're finished.";
```

Comment:
This should include the `<br>`s which exist in the actual text.
```suggestion
	private static final String TASK_EXISTING_2 = "You're still bringing balance to basilisks in the<br>Jormungand's Prison, with 136 to go. Come back when<br>you're finished.";
```

Reply:
This var name should be changed as well. It should become `TASK_CHECKSLAYERGEM_KONAR_2` and move next to that line earlier in the file.